ffmpeg/ffmpeg.go

```diff
@@ -171,6 +172,28 @@ func Transcode3(input *TranscodeOptionsIn, ps []TranscodeOptions) (*TranscodeRes
 	return t.Transcode(input, ps)
 }
 
+// Force copy-transcoding for audio-only segments
+func BypassTranscoding(input *TranscodeOptionsIn, ps []TranscodeOptions) (*TranscodeResults, error) {
```

Comment:
Could this function be private? Doesn't seem like it needs to be a part of the public API.

Reply:
Ah dang I keep forgetting capitalization => scope in golang, thanks!